src/playlists.ts

```diff
@@ -0,0 +1,37 @@
+import * as vscode from 'vscode';
+
+
+
+export class PlaylistsProvider implements vscode.TreeDataProvider<Playlist> {
+  constructor(private spotify: any) {}
```

Comment:
**Neutral:** I finally figured out how to import TypeScript definitions for the Spotify library, and doing this should make the typings for API functions a little less messy.

```suggestion
import SpotifyWebApi = require('spotify-web-api-node');

export class PlaylistsProvider implements vscode.TreeDataProvider<Playlist> {
  constructor(private spotify: SpotifyWebApi) {}
```

Reply:
Oooh gorgeous!!